frontend/src/components/Room.tsx

```diff
@@ -259,7 +259,7 @@ class Room extends React.Component<Props, State> {
     this.socket.on(Event.CONNECT, () => {
       this.socket.emit(Event.JOIN_ROOM, id);
       this.setUsernameAndEmit();
-      this.socket.emit(Event.CREATE_USERNAME, this.props.location.state.username);
+      this.socket.emit(Event.CREATE_USERNAME, this.props.location.state?.username);
```

Comment:
I think this line may have been removed in #124 to fix the same issue. @johnagl is this line unnecessary? I was noticing a weird thing where my username would be set to two different things